### PR TITLE
ecosystem-ci-artifacts: Added periodics

### PR DESCRIPTION
**Pull request by sagidayan**
Added periodics to preform e2e for 4.6 and 4.9

/hold
/cc @lranjbar 
/cc @mkowalski 

Please take a look. I know its Friday - But had to submit this PR :)

If it looks good to you guys, Please unhold it.

### CODE REVIEW

File: ci-operator/jobs/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+    repo: ci-artifacts
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws
+    ci-operator.openshift.io/variant: "4.6"
+    ci.openshift.io/generator: prowgen
+    job-release: "4.6"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-rh-ecosystem-edge-ci-artifacts-master-4.6-gpu-operator-e2e-140
+  spec:
```
Comment (kpouget) [inline ci-operator/jobs/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master-periodics.yaml:20]: slack reporting missing, I guess you'll want to enable it

File: ci-operator/jobs/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master-presubmits.yaml [hunk 2]
```diff
@@ -1,5 +1,109 @@
 presubmits:
   rh-ecosystem-edge/ci-artifacts:
+  - agent: kubernetes
+    always_run: true
```
Comment (kpouget) [inline ci-operator/jobs/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master-presubmits.yaml:4]: I don't think you want to `always_run: true` these `ci/prow/4.6-images`,
all the variants use the same image definition, only one `ci/prow/images` in the main `master` file is enough

File: ci-operator/jobs/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master-presubmits.yaml [hunk 3]
```diff
@@ -1,5 +1,109 @@
...
+      volumes:
+      - name: pull-secret
+        secret:
+          secretName: registry-pull-credentials
+      - name: result-aggregator
+        secret:
+          secretName: result-aggregator
+    trigger: (?m)^/test( | .* )4.6-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (kpouget) [inline ci-operator/jobs/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master-presubmits.yaml:56]: ditto

**Review by kpouget — COMMENTED** [review]
@sagidayan I'm surprised to see only OCP 4.6 and 4.9, what about N-1 and N-2 (4.8 and 4.7)?

slack reporting is missing for periodics, I think you'll want them
and the variants have the image built automatically, I think that's not necessary

File: ci-operator/jobs/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master-presubmits.yaml (hunk 2 above)
Comment (sagidayan) [reply ci-operator/jobs/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master-presubmits.yaml:4]: correct

**Comment by sagidayan** [comment]
@kpouget That's what was decided for now. We will add more versions later on

**Comment by kpouget** [comment]
LGTM :)

**Comment by kpouget** [comment]
cool, I'll add a config file to `ci-dashboard` on Monday :)

https://github.com/openshift-psap/ci-dashboard/blob/master/examples/gpu-operator.yml
https://openshift-psap.github.io/ci-dashboard/gpu-operator_daily-matrix.html

**Pull request merged**